scipy/special/_basic.py

```diff
@@ -3087,6 +3090,138 @@ def factorialk(n, k, exact=True):
     return _exact_factorialx_array(n, k=k)
 
 
+def stirling2(N, K, *, exact=True):
+    """Generate Stirling number(s) of the second kind.
+
+        Stirling numbers of the second kind count the number of ways to
+        partition a set with N elements into K non-empty subsets.
+
+        The values this function returns are calculated using a dynamic
+        program which avoids redundant computation across the subproblems
+        in the solution. For array-like input, this implementation also 
+        avoids redundant computation across the different Stirling number
+        calculations.
+
+        The numbers are sometimes denoted
+        .. math::
+
+            `{N \brace K}`;
```

Comment:
it looks like this line isn't getting rendered correctly:

https://output.circle-artifacts.com/output/job/8bac28a8-603b-426e-9aa6-8371b2ae3c6c/artifacts/0/html/reference/generated/scipy.special.stirling2.html

the `\b` is getting dropped somehow. This may be something implementation-wise in our sphinx setup. This renders for me in latex fine but maybe we should drop this here  notational part?

Reply:
I was looking into it. I think the docstring needs to be a raw string `r"""` instead of `"""` for the LaTeX to work correctly in all cases.